uisdk/src/main/res/layout/uisdk_alert_prebook_confirmation.xml

```diff
@@ -3,74 +3,44 @@
+        android:translationX="20dp"
+        app:layout_constraintEnd_toStartOf="@+id/rideConfirmedLogo"
+        app:layout_constraintTop_toBottomOf="@+id/masterBottomLayout"
+        app:srcCompat="@drawable/kr_fi_check_circle" />
 
-    <TextView
-        android:id="@+id/meetingPointText"
-        style="@style/Text.Faded.Medium"
-        android:layout_width="@dimen/kh_uisdk_spacing_none"
-        android:layout_height="wrap_content"
-        android:layout_marginStart="@dimen/kh_uisdk_spacing_medium_small"
-        android:layout_marginTop="@dimen/kh_uisdk_spacing_xsmall"
-        android:layout_marginEnd="@dimen/kh_uisdk_spacing_medium_small"
-        android:gravity="center_horizontal"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/rideConfirmedLabel"
-        tools:text="Placeholder text that contains information about booking with Fleet such as Meeting Points." />
-
-    <View
-        android:id="@+id/addressDivider"
-        android:layout_width="@dimen/kh_uisdk_spacing_none"
-        android:layout_height="@dimen/kh_uisdk_divider_height"
-        android:layout_marginStart="@dimen/kh_uisdk_spacing_medium_small"
+    <ImageView
+        android:id="@+id/rideConfirmedLogo"
+        android:layout_width="@dimen/kh_uisdk_logo_width_prebook_confirmation"
+        android:layout_height="@dimen/kh_uisdk_logo_height"
+        android:layout_gravity="center"
         android:layout_marginTop="@dimen/kh_uisdk_spacing_small"
-        android:layout_marginEnd="@dimen/kh_uisdk_spacing_medium_small"
-        android:background="@color/kh_uisdk_divider"
+        android:scaleType="fitCenter"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/meetingPointText" />
+        app:layout_constraintTop_toBottomOf="@+id/masterBottomLayout"
+        app:srcCompat="@drawable/uisdk_ic_prebook_confirmed" />
 
-    <TextView
-        android:id="@+id/pickupTypeText"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="@dimen/kh_uisdk_spacing_xsmall"
-        android:background="@drawable/uisdk_background_off_black_rounded"
-        android:paddingLeft="@dimen/kh_uisdk_spacing_xxsmall"
-        android:paddingTop="@dimen/kh_uisdk_spacing_xxxsmall"
-        android:paddingRight="@dimen/kh_uisdk_spacing_xxsmall"
-        android:paddingBottom="@dimen/kh_uisdk_spacing_xxxsmall"
-        android:text="@string/kh_uisdk_pickup_type_meet_and_greet"
-        android:textColor="@color/kh_uisdk_text_white"
-        android:textSize="@dimen/kh_uisdk_text_size_small"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/meetingPointText" />
 
     <TextView
```

Comment:
The address part of the layout should be extracted to a different xml in order to be able to inject it in other views as well.